ccf_transaction_processor/Makefile

```diff
@@ -25,6 +25,12 @@ ifndef SGX_MODE
 $(error Incomplete configuration, SGX_MODE is not defined)
 endif
 
+ifdef NINJA_OPTION
+NINJA_J_OPTION = $(NINJA_OPTION)
+else
+NINJA_J_OPTION = 4
+endif
```

Comment:
You could have written that way more compact as `NINJA_J_OPTION ?= 4`.

More confusing though, why do you expect `NINJA_OPTION` and then define `NINJA_J_OPTION`? If you really want to allow only override of -j, then you should also make the external variable NINJA_J_OPTION.  Personally, i would have preferred the more general NINJA_OPTIONS i had in the patch i've sent you: more versatile. Also, why are you passing it only to ninja but not, say, ninja install?